src/sql/workbench/contrib/objectExplorer/test/browser/connectionTreeActions.test.ts

```diff
@@ -79,6 +81,17 @@ suite('SQL Connection Tree Action tests', () => {
 		return connectionManagementService;
 	}
 
+	function createDialogService(choiceIndex: number): TypeMoq.Mock<IDialogService> {
```

Comment:
Add doc comments - it's not clear what "choiceIndex" will do without reading through the code.